### PR TITLE
fix: friendly error message when creating bridges with too long names

### PR DESCRIPTION
**Pull request by kjellwinblad**
This commit makes the error message and log entry that appear when one tries to create a bridge with a name the exceeds 255 bytes (the max length for atoms) more friendly and easier to understand.

An even better fix would be to not store bridge names as atoms but this probably requires a more substantial change.

Fixes:
https://emqx.atlassian.net/browse/EMQX-9609

### <samp>🤖 Generated by Copilot at 75ff76a</samp>

This pull request improves the error handling, validation, and configuration logic for bridges and rules in emqx. It also updates the version numbers of the affected applications: `emqx_rule_engine`, `emqx_ee_bridge`.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [] Added tests for the changes
- [] Changed lines covered in coverage report
- [x] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [] Schema changes are backward compatible

### CODE REVIEW

File: apps/emqx/src/emqx_config_handler.erl [hunk 1]
```diff
@@ -473,6 +473,17 @@ bin_path(ConfKeyPath) -> [bin(Key) || Key <- ConfKeyPath].
...
 bin(B) when is_binary(B) -> B.
 
+atom(Bin) when is_binary(Bin), size(Bin) > 255 ->
+    erlang:throw(
+        iolist_to_binary(
+            io_lib:format(
+                "Name is is too long."
+                " Please provide a shorter name (<= 255 bytes)."
+                " The name that is too long: \"~s\"",
+                [Bin]
```
Comment (zmstone) [inline apps/emqx/src/emqx_config_handler.erl:483]:
```suggestion
                "Name '~s' is is too long."
                " It should be less than 255 bytes."
                [Bin]
```

**Review by zmstone — COMMENTED** [review]
please add a test case for regression purposes.

File: apps/emqx/src/emqx_config_handler.erl (hunk 1 above)
Comment (kjellwinblad) [reply apps/emqx/src/emqx_config_handler.erl:483]: Originally, I had the text written more or less exactly like you suggest but then I noticed that when you have a long name the text is cut off in the log entry so all you see is "Name 'nameeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee...". That is why I changed to its current form. Do you still prefer your version?

**Comment by kjellwinblad** [comment]
> please add a test case for regression purposes.
Sure will do so.

**Review by ieQu1 — APPROVED** [review]

**Comment by yanzhiemq** [comment]
_

**Pull request merged**